modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/router/MQTTMessageRouter.java

```diff
@@ -73,7 +73,7 @@ public MQTTMessageRouter(String encodedRouterInfo) {
      * @throws AndesException
      */
     public void removeMapping(String bindingKey, StorageQueue queue) throws AndesException {
-        if (queue.getBoundedSubscriptions().isEmpty()) {
+        if (queue.getBoundSubscriptions().isEmpty()) {
```

Comment:
Can 'queue.getBoundSubscriptions()' be actually empty when this method is called?